packages/emoji-mart/src/browser.js

```diff
@@ -1,2 +1,2 @@
-import * as EmojiMart from './index.js'
+import * as EmojiMart from './index'
```

Comment:
Fun fact: the newest versions of TypeScript do now support `.js` imports! https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-7.html#ecmascript-module-support-in-nodejs

Including `.js` in the file imports is fine now. So all these `.js` removals can safely be reverted.

```suggestion
import * as EmojiMart from './index.js'
```